pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1163,6 +1363,7 @@ func handleCreate(ctx *domainContext, key string, config *types.DomainConfig) {
 
 	if config.Activate {
 		doActivate(ctx, *config, &status)
+		ctx.vmResourcesLock.Unlock()
```

Comment:
Probably I miss something, but should we unlock in case of `!config.Activate` as well?

Reply:
My idea was to unlock in 2 cases:
1. After we do a call to create a real VM, so by this moment the information is consistent. And it's done in the result of the following calls chain: `doActivate() -> doActivateTail() -> hyper.Task().Start()`.
2. In all the failure exits from the handleCreate() function.

Now, I'm curious, what is the situation, when by this moment, we have config.Active != true...

Reply:
!config.Activate may come if you deploy VM and stop it before handling on EVE-OS side (for example after reboot of EVE-OS).
Probably we can defer unlock.

Reply:
So, you mean to do it just like:
```
ctx.vmResourcesLock.Lock()
defer ctx.vmResourcesLock.Unlock()
```
?

Reply:
The lock is gone into the CPUAllocator